docs/source/index.html.md

```diff
@@ -572,6 +572,69 @@ The function returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/
 which is fulfilled with the `DataTables` instance once initialization is completed. This allows you
 all the flexibility you could need to [invoke API functions](https://datatables.net/reference/api/).
 
+# Exporting Data
+
+DataTables natively supports exporting the current view as an Excel sheet, PDF, etc. The problem with this
+is that this is all generated on the client's end meaning you'll be unable to export all data at once unless it
+happens to fit on a single screen. This is why this bundle supports exporting data from the server's end instead.
+
+## Prerequisites
+
+To make use of the exporting functionality in this bundle there are some extra dependencies you may want to install.
+These are optional as you may not need the export feature at all, or you might not need all options available.
+
+The easiest way to make the export feature work on the client side is by using the [Buttons extension](https://datatables.net/extensions/buttons/)
+offered by `DataTables`.
```

Comment:
The `Buttons` extension is required, so I would emphasize that it is a required dependency to make the server-side export feature work.

  ```suggestion
The server-side export feature requires the [Buttons extension](https://datatables.net/extensions/buttons/) offered by `DataTables`. Make sure to install it before using this feature.
```